src/components/molecules/SearchAutocomplete/interface.ts

```diff
@@ -11,3 +11,7 @@ export interface SuggestionsProps {
 export interface SuggestionProps {
   isSelected?: boolean;
 }
+
+export interface InputProps {
+  showSuggestions: boolean;
```

Comment:
I am wondering whether to indicate those are Style related props, like we did in Button case.
```suggestion
export interface InputStyleProps {
  showSuggestions: boolean;
```